src/modules/pages/NewApiPage/newApiContainer.js

```diff
@@ -4,16 +4,19 @@ import {
     fetchEndpointSchema,
     resetEndpoint,
     saveEndpoint,
+    excludePlugin,
+    selectPlugin,
     willClone,
 } from '../../../store/actions';
 
 import NewApiItem from './NewApiItem';
 
 const mapStateToProps = state => ({
     api: state.apiReducer.api,
+    selectedPlugins: state.apiReducer.selectedPlugins,
 });
 
 export default connect(
     mapStateToProps,
-    { fetchEndpointSchema, resetEndpoint, saveEndpoint, willClone },
+    { fetchEndpointSchema, resetEndpoint, saveEndpoint, excludePlugin, selectPlugin, willClone },
```

Comment:
maybe drop them to one lines for readability?

Reply:
`true`